varnish-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,936 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
```

Comment:
Actually I have a little bit of review for the dashboards, just noticed a few things. 

Would change stat panels to Graph mode: None